src/lib/mina/mina.ts

```diff
@@ -158,6 +161,11 @@ function Network(
       lightnetAccountManagerEndpoint = options.lightnetAccountManager;
       Fetch.setLightnetAccountManagerEndpoint(lightnetAccountManagerEndpoint);
     }
+
+    if (options.enforceTransactionLimits !== undefined &&
```

Comment:
I think it would be cleaner to name the option `bypassTransactionLimits`.  That way `undefined` and `false` both resolve to the default behavior.  Then I think this could be 

`enforceTransactionLimits = options.bypassTransactionLimits ? false : true;`

Reply:
I tried to reuse the same name than LocalBlockchain :
`async function LocalBlockchain({
  proofsEnabled = true,
  enforceTransactionLimits = true,
} = {}) {`

https://github.com/o1-labs/o1js/blob/a0f0f349811fd812b27cb18bac74e8edecd48f9a/src/lib/mina/local-blockchain.ts#L74